### PR TITLE
RDP header sticky when doing scroll

### PR DESCRIPTION
**Pull request by trilopin**
#### Summary
Demo https://www.loom.com/share/c211de187c91420d83db118778684aad
Keep sticky header if we scroll in RDP.
Fix width when space is lower than 780px

#### Ticket Link
No ticket

#### Checklist
- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

**Review by gbochora — APPROVED** [review]
Cool!

File: webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx [hunk 1]
```diff
@@ -189,18 +189,23 @@ const MainWrapper = styled.main<{isRHSOpen: boolean}>`
 
 const Main = styled.main<{isRHSOpen: boolean}>`
     max-width: 780px;
-    min-width: 500px;
+    width: min(780px, 100%);
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:192]: Oh I didn't know CSS had a `min` function, cool!

File: webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx [hunk 2]
```diff
@@ -180,7 +180,7 @@ const Container = styled(ColumnContainer)`
     flex: 1;
 `;
 
-const MainWrapper = styled.main<{isRHSOpen: boolean}>`
+const MainWrapper = styled.div<{isRHSOpen: boolean}>`
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:183]: Why do we need this change?

**Review by agarciamontoro — APPROVED** [review]
Looks good!

File: webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx (hunk 2 above)
Comment (trilopin) [reply webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:183]: only one main per page like tags as body, header etc..

Comment (agarciamontoro) [reply webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:183]: Oh, I see. Do we have more?

Comment (trilopin) [reply webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:183]: there is a main: real main with content (max-width:780px) and a mainwrapper (now a div) with the full width to manage the centered main

**Pull request merged**